app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -24,16 +26,27 @@
      */
     public function create(): View
     {
-        return view('auth.register');
+        $githubEmail = session()->pull('github_email');
```

Comment:
These two fields may have been set by the GitHub login process and are used in the register template